examples/tiny-process-library/boo.cpp

```diff
@@ -0,0 +1,202 @@
+#include "process.hpp"
+#include <iostream>
+
+using namespace std;
+using namespace TinyProcessLib;
+
+int main() {
+#if !defined(_WIN32) || defined(MSYS_PROCESS_USE_SH)
```

Comment:
Nice thorough tests!

Reply:
Hm, I take the [example.cpp](https://gitlab.com/eidheim/tiny-process-library/blob/master/examples.cpp) from the tiny-process-library project.
Should I remove this?!